git-open

```diff
@@ -29,6 +29,8 @@ if [ -n "$1" ]; then
       echo "'git open issue' expect branch naming to be issues/#123" 1>&2
       exit 1
     fi
+  elif [ "$1" == "pr" ]; then
+    pr="frigg yeah"
```

Comment:
Was trying to follow the pattern, but yeah it's pretty silly. Perhaps this would be better on [line 123](https://github.com/Bates550/git-open/pull/1/files#diff-21e611831fd54f9104bb1bb2abf161eeR123):
```
elif [ "$1" == "pr" ]; then
  # do pr stuff
elif [ -n $branch ]; then
  # ... 
```
```